src/components/Profile/Delegates.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react'
+import {
+  Box,
+  GU,
+  isAddress,
+  Link,
+  LoadingRing,
+  shortenAddress,
+  textStyle,
+  useTheme,
+} from '@1hive/1hive-ui'
+import IdentityBadge from '@components/IdentityBadge'
+import { useGardens } from '@providers/Gardens'
+import useUser from '@hooks/useUser'
+import { useWallet } from '@providers/Wallet'
+
+import { addressesEqual } from '@utils/web3-utils'
+import { getGardenLabel } from '@utils/garden-utils'
+
+function Delegates({ account }) {
+  const [user, loading] = useUser(account)
+
+  return (
+    <Box heading="Delegate voting" padding={0}>
+      {loading ? (
+        <div
+          css={`
+            padding: ${3 * GU}px;
+          `}
+        >
+          <LoadingRing />
+        </div>
+      ) : (
+        <div>
+          <MyDelegates account={account} user={user} />
+          <DelegateFor user={user} />
+        </div>
+      )}
+    </Box>
+  )
+}
+
+function MyDelegates({ account, user }) {
+  const theme = useTheme()
+  const { account: connectedAccount } = useWallet()
+
+  return (
+    <div
+      css={`
+        padding: ${3 * GU}px;
+      `}
+    >
+      <h3
+        css={`
+          ${textStyle('label2')};
+          margin-bottom: ${1 * GU}px;
+        `}
+      >
+        {addressesEqual(account, connectedAccount) ? 'My ' : ''}Delegates
+      </h3>
+      {user?.supports && user.supports.length > 0 ? (
+        user.supports.map((support, index) => {
+          if (!support.representative) {
+            return null
```

Comment:
since we are inside html maybe is  better to return </> instead of  null

Reply:
`null` should be the one to use when we don´t want to return anything https://reactjs.org/docs/conditional-rendering.html#preventing-component-from-rendering